arches/install/init.sql

```diff
@@ -0,0 +1,11 @@
+CREATE DATABASE template_postgis_20 WITH ENCODING 'UTF8' LC_COLLATE='en_US.utf8' LC_CTYPE='en_US.utf8';
```

Comment:
When I tried this on windows, it failed saying that the locale name was invalid. Turns out Windows encoding is completely different `English_United States.1252`. So I recommend leaving this file as is, but renaming it something like `init-unix.sql`.